src/components/ToolTip.tsx

```diff
@@ -89,8 +93,53 @@ export default function ({
   const { xs } = useBreakpoints()
   useClickOutside(childrenRef, () => setIsShow(false))
 
+  const [node, setNode] = useState<VNode | null>(null)
+
+  useEffect(() => {
+    setIsShow(!!node)
+  }, [node])
+
+  const positionTooltip = (
+    e: JSXInternal.TargetedMouseEvent<HTMLDivElement>
+  ) => {
+    if (position !== 'floating') return
+
+    const x = e.pageX
+    const y = e.pageY
+    const el = document.getElementById('root')
+    const positionX = (xs ? x * 0.5 : x * 0.95) + 'px;'
+    const positionY = y + 0.5 + 'px;'
+
+    const classLove = width('lg:w-card', 'w-6/12')
+
+    const stylestring =
+      `position:absolute; z-index: 999;` +
```

Comment:
Do we still need `z-index` if we have the portal? It will overlap the NavBar someday, because it has `z-50`

Reply:
Set it to `49`